data/json/items/armor.json

```diff
@@ -7399,6 +7410,7 @@
     "weight": 30,
     "volume": 0,
     "price": 590000,
+    "price_postapoc": 59000,
```

Comment:
That's 590 dollars, which is very different than the rest of your other changes.

Reply:
Weird, don't even remember changing that one! :s oh well, made it about 20USD.